modules/swift-syntax/510.0.0/patches/module_dot_bazel.patch

```diff
@@ -0,0 +1,13 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -1,7 +1,8 @@
+ module(
+     name = "swift-syntax",
+-    version = "0",  # Update on release/* branches
++    version = "510.0.0",
+     compatibility_level = 1,
+ )
+
+-bazel_dep(name = "rules_swift", version = "1.5.1", repo_name = "build_bazel_rules_swift")
++bazel_dep(name = "rules_swift", version = "1.16.0", repo_name = "build_bazel_rules_swift")
++bazel_dep(name = "apple_support", version = "1.11.1", repo_name = "build_bazel_apple_support")
```

Comment:
should i update these upstream? what was the issue without these bumps? (async question, not a huge deal)

Reply:
https://github.com/apple/swift-syntax/pull/2521